packages/suite/src/reducers/wallet/coinjoinReducer.ts

```diff
@@ -344,6 +344,19 @@ const updateDebugMode = (
     }
 };
 
+const enableSessionAutopause = (
+    draft: CoinjoinState,
+    payload: ExtractActionPayload<typeof COINJOIN.SESSION_AUTOPAUSE>,
+) => {
+    const session = draft.accounts.find(a => a.key === payload.accountKey)?.session;
+
+    if (!session) {
+        return;
+    }
+
+    session.isAutoPauseEnabled = payload.isAutopaused;
```

Comment:
You need to `delete` this property in `restoreSession` action, otherwise it always enables to true once it is toggled once. (The modal shows "Fine!" when it opens.)

I would also consider omitting the field from the database as we probably don't need to save across user sessions. This would be done in `serializeCoinjoinSession` in`storage.ts`.

![Screenshot 2023-03-15 at 19 15 09](https://user-images.githubusercontent.com/42465546/225404733-204ee0a8-fc31-4863-ab23-a409fd9c68fc.png)

Reply:
fd8a1d8687b8737b2ba864736a6a4f3633c9d4d6